x/stakeibc/keeper/unbonding_records.go

```diff
@@ -126,7 +133,7 @@ func (k Keeper) SendHostZoneUnbondings(ctx sdk.Context, hostZone types.HostZone)
 			sdk.NewAttribute("newAmountUnbonding", strconv.FormatUint(totalAmtToUnbond, 10)),
```

Comment:
do we need to emit this unbonding amount? Reason I'm asking is cause it feels non-ideal to have SendHostZoneUnbondings return 3 parameters instead of 2 - just to support an event